keyserver/coniksserver/main.go

```diff
@@ -0,0 +1,146 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/signal"
+	"path"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/crypto/sign"
+	"github.com/coniks-sys/coniks-go/crypto/vrf"
+	"github.com/coniks-sys/coniks-go/keyserver"
+	"github.com/coniks-sys/coniks-go/utils"
+)
+
+type boolStringFlag struct {
+	set   bool
+	value string
+}
+
+func (sf *boolStringFlag) Set(x string) error {
+	sf.value = x
+	sf.set = true
+	return nil
+}
+
+func (sf *boolStringFlag) String() string {
+	return sf.value
+}
+
+func (sf *boolStringFlag) IsBoolFlag() bool {
+	return true
+}
+
+func main() {
+	var genkey boolStringFlag
+	flag.Var(&genkey, "genkey", "Generate signing/VRF key")
```

Comment:
> genkey := flag.String("genkey", "sign,vrf", "Generate signing/VRF key")

But I want `genkey` is an optional flag, and there is no way to check if `genkey` is set or not if we use the default flag types, as I know. 

> boolStringFlag doesn't seem to be working out the way you wanted it to

Yes, that's why I have filtered the input in https://github.com/coniks-sys/coniks-go/blob/e96f8c80ec64b4edd0a02fdc368e2ac07e61f89d/keyserver/coniksserver/main.go#L103-L110.